Makefile

```diff
@@ -69,6 +73,10 @@ vet: ## Apply go vet to all go files
 build-e2e:
 	$(DOCKER_CMD) go test -c -o "$(BUILD_DEST)" github.com/openshift/cluster-api-actuator-pkg/pkg/
 
+.PHONY: unit
+unit: ## Run unit tests
```

Comment:
do we need to add a prow configuration to exercise this new target?

Reply:
Yes, this needs to be set up. The tests do pass locally right now.

Reply:
I have https://github.com/openshift/release/commit/09ee970b2de485ddfa1dda98d9628df5ab9944db prepared, but we need to merge this first. There is no unit target on master, so it won't pass until this PR adds it.